unit-tests/src/test/java/org/eclipse/collections/impl/map/fixed/AbstractMemoryEfficientMutableMapTest.java

```diff
@@ -99,6 +99,12 @@ public void removeKey_throws()
         this.classUnderTest().removeKey(null);
     }
 
+    @Test(expected = UnsupportedOperationException.class)
```

Comment:
Avoid @Test(expected)

Reply:
It is consistent with other tests in this class. I am fine using Verify.assertThrows() when there are multiple test cases in the same test method. However, here it does not make sense.